src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php

```diff
@@ -555,6 +571,16 @@ public function itsAttributeShouldBe($attributeName, ProductInterface $product,
         Assert::same($this->updateSimpleProductPage->getAttributeValue($attributeName, $language), $value);
     }
 
+    /**
+     * @Then non translatable attribute :attributeName of product :product should be :value
+     */
+    public function itsNonTranslatableAttributeShouldBe($attributeName, ProductInterface $product, $value)
```

Comment:
$attributeName and $value dont have type. Is it how this should be? And return type?